customizations/README.md

```diff
@@ -125,7 +99,7 @@ TODO: Document and add the ability to do it
     --key=path/to/key/file
   ```
 
-- [Add the following customization](#how-to-define-customizations) lines:
+- Add the following lines to you `client-conf.yaml`:
```

Comment:
I have tried to put the link to how to use the custom patches in all sections so that customers who go directly to their section have a link to where it explains what the custom patches are based on and can understand it.

Reply:
I think that adds more confusion than it helps